sgkit/window.py

```diff
@@ -12,27 +13,32 @@
 # Window definition (user code)
 
 
-def window(
+def window_by_index(
```

Comment:
Not a strong opinion here, but have you considered ``window_by_variant``? The ``index`` suffix here might refer to a number of things, and there would be a nice symmetry then between ``window_by_variant`` and ``window_by_position``.

Reply:
I hadn't thought of it, but `window_by_variant` might be better.